src/Views/Video.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+//import logo from '../logo.svg';
+import '../App.css';
+
+const BASE_EMBED_URL = 'https://www.youtube.com/embed/';
+
+class Video extends React.Component{
+
+    render() {
+        const embedUrl = `${BASE_EMBED_URL}${this.props.id}`;
+        return (
+            <div>
```

Comment:
Add some CSS to this component to make this component to enhance UI. Here's how you can do it: https://www.w3schools.com/react/react_css.asp